pkg/extensions/sync/sync.go

```diff
@@ -494,8 +496,7 @@ func Run(cfg Config, storeController storage.StoreController, wtgrp *goSync.Wait
 
 				if err := syncRegistry(regCfg, storeController, logger, localCtx, policyCtx,
 					credentialsFile[upstreamRegistry], uuid.String()); err != nil {
-					logger.Error().Err(err).Msg("sync exited with error, stopping it...")
-					ticker.Stop()
```

Comment:
Don't stop the ticker anymore since retry options can be configured.
Keep retrying on PollIntervals